perun-wui-admin/src/main/java/cz/metacentrum/perun/wui/admin/pages/perunManagement/FacilitiesManagementView.java

```diff
@@ -58,18 +69,21 @@
 		remove = PerunButton.getButton(PerunButtonType.REMOVE, ButtonType.DANGER, "Remove selected Facilities");
 		initWidget(uiBinder.createAndBindUi(this));
 
+		anchorColumnMap = new HashMap<AnchorListItem, PerunColumnType>();
+		anchorColumnMap.put(idDropdown, PerunColumnType.ID);
+		anchorColumnMap.put(nameDropdown, PerunColumnType.NAME);
+		anchorColumnMap.put(descriptionDropdown, PerunColumnType.DESCRIPTION);
+
```

Comment:
I'm missing "FACILITY_OWNERS" column here. You can see its ColumnProvider to get idea, how to get value from facility to filter on.